types/selenium-webdriver/lib/util.d.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Determines whether a {@code value} should be treated as an object.
+ * @param {?} value The value to test.
+ * @returns {boolean} Whether the value is an object.
+ */
+export function isObject(value: any): boolean;
+
+/**
+ * Determines whether a {@code value} should be treated as a promise.
+ * Any object whose "then" property is a function will be considered a promise.
+ *
+ * @param {?} value The value to test.
+ * @return {boolean} Whether the value is a promise.
+ */
+export function isPromise(value: any): boolean;
```

Comment:
```suggestion
export function isPromise(value: any): value is Promise<any>;
```
This function looks like a [type guard](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates), so might as well make it one

Reply:
Changes incorporated